test/e2e/storage/csi_volumes.go

```diff
@@ -233,12 +233,12 @@ var _ = utils.SIGDescribe("CSI Volumes", func() {
 					Prefix:    "csi-attach",
 				}
 
-				driver = drivers.InitMockCSIDriver(config, test.driverExists, test.driverAttachable, nil)
+				driver = drivers.InitMockCSIDriver(config, test.deployDriverCRD, test.driverAttachable, nil)
 				driver.CreateDriver()
+				defer driver.CleanupDriver()
 
-				if test.driverExists {
+				if test.deployDriverCRD {
 					defer destroyCSIDriver(csics, driver)
```

Comment:
Is the order relevant here?

Previously, `destroyCSIDriver` ran first, then `driver.CleanupDriver`. Now it is the other way around.

Reply:
Order doesn't entirely matter, but conceptually this new order is cleaner. We should delete the CSIDriver CRD before deleting the actual driver pods.